### ISSUE TITLE
asdl license

### ISSUE THREAD

**Issue by alexpolozov** [ElementAI/duorat#12]
Hello,

First, thanks for publishing this, great work!

The license for the entire repository is non-commercial, yet the `asdl` sub-package contains a mixture of MIT code and your additions (presumably under the same non-commercial license). Would it be possible to re-license at least the `asdl` utility package as MIT? It (a) already builds upon open-source MIT code, (b) is not the core contribution of the project. That would be much appreciated by the community, and would help referencing this work and building upon it.

**Comment by rizar** [comment]
Hi Alex, 

Let us see what we can do. Element AI has recently become a part of ServiceNow, and we'll have to adhere to our new open-source processes.

Would Apache license resolve your concerns, or do you think MIT is the only way to go?

**Comment by alexpolozov** [comment]
Yes, Apache would work too, the question is about commercial code licensing rather than IP. Thank you, Dzmitry!
